gst/nnstreamer/include/nnstreamer_plugin_api_filter.h

```diff
@@ -157,8 +157,8 @@ typedef struct _GstTensorFilterFrameworkInfo
 {
   const char *name; /**< Name of the neural network framework, searchable by FRAMEWORK property. Subplugin is supposed to allocate/deallocate. */
   int allow_in_place; /**< TRUE(nonzero) if in-place transfer of input-to-output is allowed. Not supported in main, yet. */
-  int allocate_in_invoke; /**< TRUE(nonzero) if invoke_NN is going to allocate output ptr by itself and return the address via output ptr. Do not change this value after cap negotiation is complete (or the stream has been started). */
-  int run_without_model; /**< TRUE(nonzero) when the neural network framework does not need a model file. Tensor-filter will run invoke_NN without model. */
+  int allocate_in_invoke; /**< TRUE(nonzero) if invoke[_NN] is going to allocate output ptr by itself and return the address via output ptr. Do not change this value after cap negotiation is complete (or the stream has been started). */
+  int run_without_model; /**< TRUE(nonzero) when the neural network framework does not need a model file. Tensor-filter will run invoke[_NN] without model. */
```

Comment:
Why have you changed this?

Reply:
I believe that the function, to which is referred, is called 'invoke_NN' in GST_TENSOR_FILTER_FRAMEWORK_V0, and 'invoke' GST_TENSOR_FILTER_FRAMEWORK_V1. I can revert, as this is not relevant to this PR.